dev/io.openliberty.jaxws.globalhandler.internal/src/com/ibm/ws/jaxws/globalhandler/GlobalhandlerSOAPMessageContextImpl.java

```diff
@@ -10,6 +10,7 @@
  *******************************************************************************/
 package com.ibm.ws.jaxws.globalhandler;
 
+
```

Comment:
This java file does not need to be included in this PR if all you are doing is adding spaces.